### ISSUE TITLE
A possible misleading comment.

### ISSUE THREAD

**Issue by zackmawaldi** [RosettaCommons/RFDesign#42]
https://github.com/RosettaCommons/RFDesign/blob/bba6992283de63faba6ff727bb4bc68327a5356c/hallucination/hallucinate.py#L109

https://github.com/RosettaCommons/RFDesign/blob/bba6992283de63faba6ff727bb4bc68327a5356c/hallucination/loss.py#L680

Looking at how the code works, seems like -1 is to turn off, not 1. That's unless I'm understanding the code wrong (which is possible!).

**Comment by jueseph** [comment]
Yes, you're right, there is a typo in the comment. It seems there is an
equals sign where a minus/hyphen should be.
